src/js/components/UploadForm.jsx

```diff
@@ -30,52 +26,96 @@ export const renderErrors = (errors) => {
 }
 
 export const getDropzoneText = ({ code, errors, filename }) => {
-  let howToMessage = 'To begin uploading a file, drag it into this box or click here.'
-  if(code >= CREATED) {
-    howToMessage = 'To begin uploading a new file, drag it into this box or click here.'
+  let howToMessage =
+    'To begin uploading a file, drag it into this box or click here.'
+  if (code >= STATUS.CREATED) {
+    howToMessage =
+      'To begin uploading a new file, drag it into this box or click here.'
   }
   let message = <p>{howToMessage}</p>
 
-  if(code >= UPLOADING) {
+  if (code >= STATUS.UPLOADING) {
     message = howToMessage
   }
 
-  if(code === SIGNED) {
-    message = <div>
-      <p>Your submission is complete.</p>
-      <p className="file-selected">{howToMessage}</p>
-    </div>
-  }
+  if (filename) {
```

Comment:
I think there should be a helper function that returns the contents of the message in each case, something like:

```
const getUploadMessage = (pretext, filename, posttext) =>
  return (
    <div>
      <p>
        {pretext}<strong>{filename}</strong>{posttext}
      </p>
      <p className="file-selected">{howToMessage}</p>
    </div>
  )
)